Sources/swift-bundler/Bundler/SwiftPackageManager/Platform.swift

```diff
@@ -71,3 +71,48 @@ enum Platform: String, CaseIterable {
     #endif
   }
 }
+
+/// An apple SDK platform name to build for.
+enum AppleSDKPlatform: String, CaseIterable {
```

Comment:
Oh yeah, sorry I didn't realise this on the first review, I got distracted 😭 Just move `AppleSDKPlatform` to a separate file and then it should all be good to merge.

Reply:
@stackotter The only reason I kept them in the same file is because you'll typically want to keep these 2 enums in sync across any changes, and wasn't sure if having them in the same file would make it any easier in the future.

But no problem, I'll move it to its separate file.